web/src/pages/User/Login.tsx

```diff
@@ -73,6 +73,12 @@ const Page: React.FC = () => {
     });
   };
 
+  const onKeyDown = (e: any) => {
```

Comment:
Please use the correct TS typing.

```suggestion
  const onKeyDown = (e: React.KeyboardEvent) => {
```